libr/anal/p/anal_sbpf.c

```diff
@@ -120,8 +132,25 @@ static RList *sbpf_find_string_xrefs(RAnal *anal, ut64 from, ut64 to, ut64 data_
 			continue;
 		}
 
+		ut64 imm_val = 0;
+		bool found_pattern = false;
+
+		// Check for MOV + HOR64 pattern (v2+ sBPF: opcodes 0xb4 or 0xb7 followed by 0xf7)
+		if ((buf[0] == SBPF_INS_MOV_IMM || buf[0] == SBPF_INS_MOV64_IMM) && buf[8] == SBPF_INS_HOR64) {
```

Comment:
```suggestion
		const ut8 b0 = buf[0];
		const bool ismov = (b0 == SBPF_INS_MOV_IMM || b0 == SBPF_INS_MOV64_IMM);
		if (ismov && buf[8] == SBPF_INS_HOR64) {
```

more readable this way